### PR TITLE
Rename all si_funcs to evt_funcs

### LINKED ISSUE #397: Rename all the si funcs to evt

### PR DESCRIPTION
**Pull request by Brotenko**
resolve #397﻿

### CODE REVIEW

File: include/PR/rcp.h [hunk 1]
```diff
@@ -1,885 +1,885 @@
...
+#define RI_RERROR_REG		(RI_BASE_REG+0x18)
+
+/* RI error (W): [] any write clears all error bits */
+#define RI_WERROR_REG		(RI_BASE_REG+0x1C)
+
+
+/*************************************************************************
+ * Serial Interface (SI) Registers
+ */
+#define EVT_BASE_REG		0x04800000
```
Comment (ethteck) [inline include/PR/rcp.h:821]: All changes to this file should be reverted since this is libultra

**Review by ethteck — CHANGES_REQUESTED** [review]

**Comment by BowserSlug** [comment]
✅ This PR fixes 1 warnings!

**Comment by Brotenko** [comment]
On @ethteck's request I will also rename every file in `evt.c` to have the `evt_` prefix

**Comment by nanaian** [comment]
We should merge the other two open PRs before this one IMO & then ideally we can reapply the big rename to the new matches

**Review by ethteck — APPROVED** [review]

**Pull request merged**